dist/index.js

```diff
@@ -299,14 +300,16 @@ class IssuesProcessor {
             else {
                 this._logger.info(`${logger_service_1.LoggerService.yellow('Processing the batch of issues')} ${logger_service_1.LoggerService.cyan(`#${page}`)} ${logger_service_1.LoggerService.yellow('containing')} ${logger_service_1.LoggerService.cyan(issues.length)} ${logger_service_1.LoggerService.yellow(`issue${issues.length > 1 ? 's' : ''}...`)}`);
             }
+            const labelsToAddWhenUnstale = words_to_list_1.wordsToList(this.options.labelsToAddWhenUnstale);
```

Comment:
Is it reasonable to add this var as a global and have it set once via the constructor? I'm not a fan of changing the args to `processIssue` to pass labels to add/remove when unstale. Not the most familiar with best practices in ts though.